README.md

```diff
@@ -63,14 +99,20 @@ CREATE TABLE report (
 -- -1 is dislike
 CREATE TABLE cluster_rule_user_feedback (
     cluster_id VARCHAR NOT NULL,
-    rule_id INTEGER  NOT NULL,
+    rule_id VARCHAR NOT NULL,
```

Comment:
that's interesting how it worked before?

Reply:
Because we use sqlite and don't test on postgres ;)

Reply:
This is from the new recent migration where the schema was altered, just the readme wasnt updated, so it was as OK then

Reply:
oh, actually no, the migration is corrent, only readme was outdated

Reply:
> Because we use sqlite and don't test on postgres ;)

yeah that's Visual Basic approach with its stupid `VARIANT` type